<p>Binary operators should always be separated by spaces.</p>

<h2>Noncompliant Code Example</h2>
<pre>
	WHEN versionRef<1.2 THEN
</pre>
<h2>Compliant Solution</h2>
<pre>
	WHEN versionRef < 1.2 THEN
</pre>
