<p>A blank space should follow each comma in any ESQL statement that makes use of commas outside of a string literal.</p>

<h2>Noncompliant Code Example</h2>
<pre>
	DECLARE myInt,myInt3 INTEGER 1;
</pre>
<h2>Compliant Solution</h2>
<pre>
	DECLARE myInt, myInt3 INTEGER 1;
</pre>