<p>Duplicated string literals make the process of refactoring error-prone, since you must be sure to update all occurrences.</p>
<p>On the other hand, constants can be referenced from many places, but only need to be updated in a single place.</p>
<h2>Noncompliant Code Example</h2>
<p>With the default threshold of 3:</p>
<pre>
CREATE PROCEDURE  run()
BEGIN
  prepare('action1');                              -- Noncompliant - 'action1' is duplicated 3 times
  execute('action1');
  release('action1');
END;
</pre>
